### DISCUSSION TITLE
marker cluster group in react-leaflet

### DISCUSSION THREAD

**Discussion by mohsenghalandar** [Leaflet/Leaflet#7995] [General]
according to leaflet docs - I just need to change LayerGroup to MarkerClusreGroup BUT in react we don't have MarkerclusretGroup

how can I use cluster group feature in my react-app with react-leaflet ?

**Comment by Falke-Design** [comment]
You need to import the Plugin [MarkerCluster](https://github.com/Leaflet/Leaflet.markercluster) it is not in Leaflet core implemented.

I found [this](https://www.npmjs.com/package/react-leaflet-markercluster) react version, maybe this works better for you.
